frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx

```diff
@@ -13,12 +14,42 @@ const generateClassTag = (rawValue: string): string => {
 };
 
 const StatusCell = ({ value }: IStatusCellProps): JSX.Element => {
+  console.log(`value: ${value}`);
   const statusClassName = classnames(
     "data-table__status",
     `data-table__status--${generateClassTag(value)}`
   );
+  const tipContent = (value: string): string | undefined => {
+    switch (value) {
+      case "online":
+        return "Online hosts will respond to a live query.";
+      case "offline":
+        return `Offline hosts won't respond to a live query because<br/>
+                they may be shut down, asleep, or not connected to<br/>
+                the internet.`;
+    }
+  };
 
-  return <span className={statusClassName}>{value}</span>;
+  // TODO: get unique id for each host
+  const id = "TESTID";
```

Comment:
does this need to be completed? something with a more meaningful value such as the host id?